### ISSUE TITLE
Twiggy error not being displayed in log file

### ISSUE THREAD

**Issue by mqwilber** [jkitzes/macroeco#98]
Run the following (incorrect) parameter file with MacroecoDesktop. The terminal is catching the error, but it is not being displayed in the log file

```
[SAD ANBO Row > 2]
analysis = sad
metadata = ANBO.txt

cols = spp_col:spp;count_col.counts
subset = row>=2
splits = row:2; column:2
log_y = True
```

**Comment by mqwilber** [comment]
Added a partial fix 1874def. A simple verification function checks the parameter file and logs where the parameter string might be mis-formatted.  twiggy is still throwing an internal error though, so it looks like the analysis stalls.

**Comment by jkitzes** [comment]
Closed, with quite a bit of work, in a7f518172375312c3f62cdeede76dc5ce9d2481a. I ended up removing the twiggy dependency all together (takes away one more possible problem in installation, see #92), and using the built in logging module instead. In doing so, this problem was fixed.
